### ISSUE TITLE
Offer ES6 exports

### ISSUE THREAD

**Issue by dilyanpalauzov** [juliendargelos/rollup-plugin-html-minifier#2]
I put in `package.json` : `"type":"module"` and I use the rollup API 
```javascript
import htmlMinifier from 'rollup-plugin-html-minifier'

rollup({
   input: '...js',
   plugins: [
       htmlMinifier.default({collapseWhitespace: true})
   ]
}).then(p => p.write(…))
```

Please provide ES6 exports, so that I do not have to write `htmlMinifier.default()`'.

This will likely need `exports` clause in `package.json`.

**Comment by juliendargelos** [comment]
I will fix that very soon.

**Comment by juliendargelos** [comment]
Fixed in v1.1.0 which uses rollup to improve plugin build. A `"module"` field now exposes an es6 bundle in `package.json`.
